report_builder/api/views.py

```diff
@@ -147,10 +147,10 @@ def post(self, request):
                         'field_verbose': field,
                         'field_type': 'Property',
                         'field_choices': None,
-                        'can_filter': True if filters is None or new_field.name in filters else False,
+                        'can_filter': True if filters is None or field in filters else False,
```

Comment:
@grantjenks I tried to copy the `name` field. For this one:

`'name': field,`. I thought it should mirror how the `name` field was set.